app/models/job.rb

```diff
@@ -0,0 +1,8 @@
+class Job < ActiveRecord::Base
+  
+  scope :title, -> (title) { where title: title }
+  scope :location, -> (location) { where location: location }
+  scope :company, -> (company) { where company: company }
+  scope :category, -> (category) { where category: category }
```

Comment:
remove scopes. you need this only in one place. so, build it just live filter options